pegasus/sites.v3/code.org/i18n/views/hoc2022_create_activities.ms-MY.md.partial

```diff
@@ -1,7 +1,7 @@
-{{ featured_tutorial, title: "Dance Party", body: "Code a Dance Party! Featuring Beyoncé, Harry Styles, Lizzo, Lil Nas X, Selena Gomez, music from Disney's \"Encanto,\" and more!", image: "/images/hoc2022-dance-party.png", url: "https://code.org/dance" }}
+{{ featured_tutorial, title: “Parti menari”, body: “Kod pesta Tarian! Menampilkan Beyoncé, Harry Styles, Lizzo, Lil Nas X, Selena Gomez, muzik daripada Disney's \"Encanto,\" dan banyak lagi!", image: "/images/hoc2022-dance-party.png", url: "https://code.org/dance" }}
 
-{{ featured_tutorial, title: "Poem Art", body: "Tambah seni dan animasi ke puisi klasik dalam aktiviti Hour of Code ini!", image: "/shared/images/courses/logo_tall_poem-art.jpg", url: "https://studio.code.org/s/poem-art-2021/reset" }}
+{{ featured_tutorial, title: “Poem Art”, body: “Tambah seni dan animasi ke puisi klasik dalam aktiviti Hour of Code ini!” , image: "/shared/images/courses/logo_tall_poem-art.jpg “, url:" https://studio.code.org/s/poem-art-2021/reset " }}
 
-{{ featured_tutorial, title: "Artist", body: "Lukis gambar dan reka bentuk yang sejuk dengan Artis!", image: "/shared/images/courses/logo_tall_artist.jpg", url: "https://studio.code.org/s/artist/reset" }}
+{{ featured_tutorial, title: “Artist”, body: “Lukis gambar dan reka bentuk yang sejuk dengan Artis!” , image: "/shared/images/courses/logo_tall_artist.jpg “, url:" https://studio.code.org/s/artist/reset " }}
 
-{{ featured_tutorial, title: "Infinity Play Lab", body: "Gunakan Play Lab untuk membuat cerita atau permainan yang dibintangi oleh watak Disney Infinity.", image: "/shared/images/courses/logo_tall_infinity.jpg", url: "https://studio.code.org/s/infinity/reset" }}
+{{ featured_tutorial, title: “Infinity Play Lab”, body: “Gunakan Play Lab untuk mencipta cerita atau permainan yang dibintangi oleh watak Disney Infinity.“, image:" /shared/images/courses/logo_tall_infinity.jpg“, url:" https://studio.code.org/s/infinity/reset" }}
```

Comment:
We have to worry about these partials and ensuring they use the right quotes... this would break the card for each mistake and possibly the rest of the cards from the point of the first mistake. I've been manually repairing these while the deploy was possible. @mgc1194 @castro-jorge